README.md

```diff
@@ -27,6 +27,17 @@ Add one line per compiler version you want to test. `latest` is the latest
 stable version of OCaml. The `[...]` are other environments variables set for
 this Travis CI run.
 
+### Changing the Base OPAM Remote
+
+```shell
+env:
+  - [...] BASE_REMOTE=<url> [...]
+```
+
+The bare-bones install script can be configured to initialize OPAM with
+a metadata repository that isn't the default community OPAM remote of
+[git://github.com/ocaml/opam-repository](git://github.com/ocaml/opam-repository).
+`BASE_REMOTE` initializes OPAM with a repository address of your choice.
```

Comment:
very useful! It would also be nice to be able to add additional remotes (e.g. mirage-dev or ocsigen-dev)